docs/docs/recipes/integrate-logto/next-js.mdx

```diff
@@ -56,6 +59,7 @@ import LogtoClient from '@logto/next';
 
 export const logtoClient = new LogtoClient({
   appId: '<your-application-id>',
+  appSecret: '<your-app-secret-copied-from-console>',
```

Comment:
The `appSecret` is an optional config, right? This PR makes it look like it's required

Reply:
For traditional web app, it IS required, without `appSecret`, the grant will fail.